Intermittent Test Failure

We are seeing an intermittent test failure:

```
go test -v github.com/[secure]/[secure]-p2p/internal -coverprofile=./build/internal.out -coverpkg=./internal/...
=== RUN   TestSyncNoError
--- PASS: TestSyncNoError (3.02s)
=== RUN   TestSyncChainID
2021-07-07T19:51:07.610Z	ERROR	p2p-gorpc	go-libp2p-gorpc@v0.1.2/call.go:63	msgpack decode error [pos 0]: stream reset
    p2p_test.go:265: Never got expected error
--- FAIL: TestSyncChainID (2.03s)
=== RUN   TestApplyBlockFailure
--- PASS: TestApplyBlockFailure (0.81s)
FAIL
coverage: 51.1% of statements in ./internal/...
FAIL	github.com/[secure]/[secure]-p2p/internal	5.871s
FAIL
```

--- a/internal/p2p_test.go
+++ b/internal/p2p_test.go
@@ -253,10 +253,10 @@ func TestSyncChainID(t *testing.T) {
 		time.Sleep(time.Duration(100) * time.Duration(time.Millisecond))
 		listenError, hasListenError := listenNode.ConnectionManager.Blacklist.GetBlacklistEntry(sendNode.Host.ID())
 		sendError, hasSendError := sendNode.ConnectionManager.Blacklist.GetBlacklistEntry(listenNode.Host.ID())
-		if hasListenError &&
-			hasSendError &&
-			strings.HasSuffix(listenError.Error.Error(), "peer's chain id does not match") &&
-			strings.HasSuffix(sendError.Error.Error(), "peer's chain id does not match") {
+		if hasListenError && strings.HasSuffix(listenError.Error.Error(), "peer's chain id does not match") {
+			ok = true
+			break
+		} else if hasSendError && strings.HasSuffix(sendError.Error.Error(), "peer's chain id does not match") {
 			ok = true
 			break
 		}

